package.json

```diff
@@ -25,7 +25,7 @@
     "@bazel/concatjs": "5.8.1",
     "@bazel/esbuild": "5.8.1",
     "@bazel/protractor": "5.8.1",
-    "@bazel/runfiles": "5.8.1",
+    "@bazel/runfiles": "6.3.1",
```

Comment:
Is this safe to this major? Or it should be ignored cross-repo?

Reply:
I don't 100% know. I think it's fine though and just supports newer versions